modules/RTC/TraceablePeerConnection.js

```diff
@@ -1869,24 +1959,9 @@ TraceablePeerConnection.prototype.setLocalDescription = function(description) {
 
     this.trace('setLocalDescription::preTransform', dumpSDP(localSdp));
 
-    if (this.options.disableH264 || this.options.preferH264) {
-        const parsedSdp = transform.parse(localSdp.sdp);
-        const videoMLine = parsedSdp.media.find(m => m.type === 'video');
-
-        if (this.options.disableH264) {
-            SDPUtil.stripVideoCodec(videoMLine, 'h264');
-        } else {
-            SDPUtil.preferVideoCodec(videoMLine, 'h264');
-        }
-
-        localSdp = new RTCSessionDescription({
-            type: localSdp.type,
-            sdp: transform.write(parsedSdp)
-        });
-
-        this.trace('setLocalDescription::postTransform (H264)',
-            dumpSDP(localSdp));
-    }
+    // Munge the order of the codecs based on the preferences set through config.js
+    // eslint-disable-next-line no-param-reassign
+    description = this._mungeCodecOrder(localSdp);
```

Comment:
`localSdp = this._mungeCodecOrder(localSdp);` and the eslint-disable is not necessary here

Reply:
I'm not sure how this ever worked given that `localSdp` instead of the `description` is used by all of the code below

Reply:
Thanks for pointing this out, I think this bug crept in when I moved the logic out to mungeCodecOrder 🤦